include/alpaka/block/shared/st/BlockSharedMemStOmp5BuiltIn.hpp

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2022 Jeffrey Kelling
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#if(defined(ALPAKA_ACC_ANY_BT_OMP5_ENABLED) || defined(ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENBLED))                           \
```

Comment:
Typo:

```suggestion
#if(defined(ALPAKA_ACC_ANY_BT_OMP5_ENABLED) || defined(ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLED))                           \
```

Also, why is the OpenMP 2 back-end tested here?

Reply:
Initially I also considered also testing these smem implementations with this backend. This change should not be in this PR, though: Originally I placed my test case in this file and then these the existing test cases triggered compiler bugs on their own. With the new test case being separate this work-around is not necessary.

Reply:
These implementations could in theory be used for `Omp2Threads` and may be better then `BlockScharedMem*Member`, depending on what the compiler and runtime are doing, so I prefer to enable them for Omp2Threads, too.